os/arch/arm/src/armv7-a/arm_prefetchabort.c

```diff
@@ -60,6 +60,10 @@
 #ifdef CONFIG_APP_BINARY_SEPARATION
```

Comment:
Minor comment, few typos in commit message excuted, pretech

Reply:
thank you I fixed to #6654